rstudio/rstudio.sh

```diff
@@ -68,9 +68,11 @@ function get_apt_key_for_debian() {
 }
 
 if [[ "${ROLE}" == 'Master' ]]; then
-  if (( ${#USER_PASSWORD} < 7 )) ; then
-    echo "You must specify a password of at least 7 characters for user \`$USER_NAME\` through metadata \`rstudio-password\`."
-    exit 1
+  if [[ ! -z ${USER_PASSWORD} ]] ; then
+    if (( ${#USER_PASSWORD} < 7 )) ; then
+      echo "You must specify a password of at least 7 characters for user \`$USER_NAME\` through metadata \`rstudio-password\`."
```

Comment:
Maybe use regular single quotes instead, to avoid escaping of back-quotes?

Reply:
Done